libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -412,6 +429,130 @@ void AP_Mount_Backend::write_log(uint64_t timestamp_us)
     AP::logger().WriteCriticalBlock(&pkt, sizeof(pkt));
 }
 
+#if AP_MOUNT_POI_TO_LATLNGALT_ENABLED
+// send camera field of view status
+void AP_Mount_Backend::send_camera_fov_status(mavlink_channel_t chan)
```

Comment:
This should live in Camera.  We need a `bool AP_Mount_Backend::get_poi(Quaternion &quat, Location &loc)` instead of this method